andela/AndelaEatsUI
# #162075277 Suspend a vendor

#### What does this PR do?
This PR suspends a vendor

#### Description of Task to be completed?
Instead of permanently deleting a vendor, admin can suspend a vendor temporary.

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-admin-view-suspended-vendors-162038590`
- run these commands `npm run bundle` and `npm run dev` 
- Log in as an `Admin`, navigate to `admin dashboard`, select `Vendor Icon`, on the sidebar and click on `Vendors`. Suspend a vendor and the suspended vendor will be added to the `Suspended Vendors` list
- run the command `npm run test-dev` to ensure all tests are passing

#### What are the relevant pivotal tracker stories?
- [#162075277](https://www.pivotaltracker.com/n/projects/2178749/stories/162075277)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1277" alt="screen shot 2018-11-20 at 1 52 52 pm" src="https://user-images.githubusercontent.com/21138774/48775120-682c0900-eccc-11e8-999b-0c8359caa2b7.png">
<img width="1273" alt="screen shot 2018-11-20 at 1 53 09 pm" src="https://user-images.githubusercontent.com/21138774/48775155-8b56b880-eccc-11e8-8ed3-03a9641dae14.png">
<img width="1280" alt="screen shot 2018-11-20 at 1 53 14 pm" src="https://user-images.githubusercontent.com/21138774/48775184-9d385b80-eccc-11e8-968d-95c1614cffbd.png">
